core/cli/cli.js

```diff
@@ -43,10 +44,10 @@ if (typeof argValue === 'undefined') {
     process.exit(1);
 }
 
-function start() {
+function start(prod) {
     const {opts, cwd} = get_cli_args();
```

Comment:
We can get rid of that line and of the whole `get_cli_args` function. We would then set `const cwd = process.cwd()`